bin/tag-release-continue.js

```diff
@@ -8,13 +8,17 @@ const utils = require("../src/utils.js");
 const filterFlowBasedOnDevelopBranch = require("../src/helpers/filterFlowBasedOnDevelopBranch");
 const getCurrentBranch = require("../src/helpers/getCurrentBranch");
 const runWorkflow = require("../src/helpers/runWorkflow");
+const { runPostScript } = require("../src/workflows/steps/index.js");
 
 utils.applyCommanderOptions(commander);
 
 commander.parse(process.argv);
 
 const callback = async options => {
 	options.callback = () => console.log("Finished"); // eslint-disable-line no-console
+	if (options.scripts[`post${options.command}`]) {
```

Comment:
how is flow and options.workflow different?

Reply:
That piece shouldn't have been there. As for if they are different, not really. 😄 But I can't remember if we didn't overwrite the workflow off the options for a reason or not off the top of my head.